README.md

```diff
@@ -2,19 +2,17 @@
 
 [![Build Status](https://travis-ci.org/github/explore.svg?branch=master)](https://travis-ci.org/github/explore)
 
-<img src="https://user-images.githubusercontent.com/1840802/31236555-436777c4-a9a9-11e7-8377-c5c4c4c7722e.png" width="650" />
+This repository houses all of the community-curated content for GitHub Topics and Collections.
 
-Topic pages, curated by the community.
+[Topics](https://help.github.com/articles/about-topics/) help you explore repositories in a particular subject area, learn more about that subject, and find projects to contribute to.
 
-[Topics](https://help.github.com/articles/about-topics/) help you explore repositories in a particular subject area, learn more about a subject area, and find projects to contribute to.
+[Collections](http://github.com/collections) help you discover hand-picked repositories, developers, organizations, videos, and articles that share a common theme.
 
-If you want to suggest edits to an existing topic page, or curate a new topic page, read our [contributing guide](CONTRIBUTING.md) to get started. You can [review a list of popular topics that need more context](topics-todo.md) to get an idea of where to start.
+If you want to suggest edits to an existing topic page or collection, or curate a new one, read our [contributing guide](CONTRIBUTING.md) to get started. You can also [review a list of popular topics that need more context](topics-todo.md) to get an idea of where to start.
 
 ## Running tests
 
-There are some lint tests in place to ensure each topic is formatted in the way we expect. Travis
-CI will run the tests automatically. If you want to run the tests yourself locally, you will need
-Ruby and Bundler installed.
+There are some lint tests in place to ensure each topic is formatted in the way we expect. Travis CI will run the tests automatically. If you want to run the tests yourself locally, you will need Ruby and Bundler installed.
```

Comment:
Should this also mention collections? Maybe not yet if there aren't any automated tests for collection formatting.

Reply:
+1. Shall we add a test in the same vein as the [`test/topics_test.rb`](https://github.com/github/explore/blob/collections/test/topics_test.rb)?

Reply:
I will work on adding a test for collections